include/collision_check.h

```diff
@@ -86,9 +86,14 @@ typedef struct ColliderElementDamageInfoAT {
     /* 0x05 */ u8 damage; // Damage
 } ColliderElementDamageInfoAT; // size = 0x08
 
+typedef enum HitBacklash {
+    HIT_BACKLASH_0,
+    HIT_BACKLASH_1 // Shock?
+} HitBacklash;
+
 typedef struct ColliderElementDamageInfoAC {
     /* 0x00 */ u32 dmgFlags; // Damage types that may affect this collider element as AC.
-    /* 0x04 */ u8 effect;  // Damage Effect (Knockback, Fire, etc.)
+    /* 0x04 */ u8 hitBacklash;  // The backlash type applied to other ATs touching this AC.
```

Comment:
Similarly something like "The hit backlash type applied to the attacker when hurting this AC collider"